pages/dao/[symbol]/proposal/components/instructions/Solend/CreateObligationAccount.tsx

```diff
@@ -39,8 +47,21 @@ const CreateObligationAccount = ({
     return <>This instruction does not support {connection.cluster}</>;
   }
 
-  // only need governance select for this instruction
-  return null;
+  return (
+    <Select
+      label="Lending Market"
+      value={form.lendingMarketName}
+      placeholder="Please select..."
+      onChange={(value) =>
+        handleSetForm({ value, propertyName: 'lendingMarketName' })
+      }
+      error={formErrors['baseTokenName']}
```

Comment:
I think it should be `formErrors['lendingMarketName']`, probably my mistake ^^'

Reply:
I suspect more some copy/paste gone wrong.